test/compatibility/common/compatibility_config.py

```diff
@@ -28,13 +28,49 @@
 
 # Common configuration for compatibility tests.
 
+import os
+import re
+
+from compatibility_version import WTVersion
+from typing import List
+
 # The branches we use for the testing. We support special branch name 'this' that refers to the
 # current branch. This is useful when debugging a compatibility issue on the current branch, but it
 # should not be enabled when testing on Evergreen.
-BRANCHES = ['develop', 'mongodb-7.1', 'mongodb-7.0', 'mongodb-6.3']
+
+# To make this branches compatitable with existing "compatibility_test_for_releases.sh", the
+# version is imported from the bash file of "meta/versions.sh"
+
+class WTBranches:
+
+    def __init__(self, branches:List[str] = []):
+        # The branches here aims to support non-script mode
+        self.SUITE_RELEASE_BRANCHES = [WTVersion(branch) for branch in branches]
+
+    def extract_versions(self, bash_script:str):
+        with open(bash_script, "r") as f:
+            content = f.read()
+        for name in ['SUITE_RELEASE_BRANCHES']:
+            match = re.search(r'export\s+'+name+r'\s*=\s*"([^"]*)"', content)
```

Comment:
This regex is a bit cryptic. Can a comment be added to explain it (e.g. with an example match)?